config/rbac/role.yaml

```diff
@@ -17,6 +17,7 @@ rules:
 - apiGroups:
   - openstack.k-orc.cloud
   resources:
+  - Keypairs
```

Comment:
Should be lowercase. I suspect this is the result of an experimentation with the scaffolding tool.